lib/remote_node_indexer.go

```diff
@@ -1,46 +0,0 @@
-package lib
```

Comment:
I could have sworn we deleted this in an earlier PR. Not sure why it's showing up here but hopefully I didn't mess anything up with my branch.